packages/manager-react-components/src/hooks/date/useFormatDate.ts

```diff
@@ -0,0 +1,78 @@
+import { useCallback } from 'react';
+import { format as formatDateFns, isValid, Locale } from 'date-fns';
+import { useTranslation } from 'react-i18next';
+import { enGB, fr, frCA, de, es, it, pl, pt } from 'date-fns/locale';
+import { getDateFnsLocale } from '@ovh-ux/manager-core-utils';
+
+export const DEFAULT_UNKNOWN_DATE_LABEL = 'N/A';
+
+const localeMap = {
+  enGB,
+  fr,
+  frCA,
+  de,
+  es,
+  it,
+  pl,
+  pt,
+} as const;
+
+export const useDateFnsLocale = () => {
+  const { i18n } = useTranslation();
+  const language = getDateFnsLocale(i18n?.language);
+  return localeMap[language] || enGB;
+};
```

Comment:
Not the right hook to export.
```suggestion 
const useDateFnsLocale = () => {
  const { i18n } = useTranslation();
  const language = getDateFnsLocale(i18n?.language);
  return localeMap[language] || enGB;
};
```

Reply:
This hook can be really useful for other DateFns usage. May be place in another place but if we can export it it's a bonus I think